src/libMessage/ZilliqaMessage.proto

```diff
@@ -264,10 +270,11 @@ message ProtoStateIndex
 
 message ProtoStateData
 {
-    optional string vname = 1; // Added in: v1.0, Deprecated in: N/A
-    optional bool ismutable = 2; // Added in: v1.0, Deprecated in: N/A
-    optional string type  = 3; // Added in: v1.0, Deprecated in: N/A
-    optional string value = 4; // Added in: v1.0, Deprecated in: N/A 
+    required uint32 version = 1; // Added in: v1.0, Deprecated in: N/A
```

Comment:
For consistency with other databases, should we change version to optional?

Reply:
I think version should be required, we can update other databases

Reply:
Ok, but everything else should be optional I think.  Please make the update to the other databases as well.